src/components/MUITypography/Typography.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { createMuiTheme, ThemeProvider, Typography } from '@material-ui/core';
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ProgressButton } from '../ProgressButton';
+
+export default { title: 'Components/Typography' };
+
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: 'Arial',
+    button: {
+      fontSize: 10,
+    },
+  },
+});
+
+export const Story = () => (
+  <ThemeProvider theme={theme}>
```

Comment:
Storybook is already using our theme. No need to redefine it here, or change it.